contrib/upload-kktix-ticket-csv-to-bigquery.py

```diff
@@ -322,5 +322,63 @@ def test_hash_email(self):
         )
 
 
+class Test2019Ticket(unittest.TestCase):
+    """python -m unittest upload-kktix-ticket-csv-to-bigquery.py"""
```

Comment:
💯  LGTM, my comment is the same as <https://github.com/pycontw/PyCon-ETL/pull/36#pullrequestreview-688404781>

Reply:
Thanks for the review!